certified-connectors/AutoPylot Connect/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "60mOMrhl23bdJmprB0pDzrLF7MXCqyzHMTitaOZeU7E",
+          "scopes": [
+            "public write"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://autopylot.com/oauth/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://autopylot.com/oauth/token"
+            },
+            "refreshUrl": {
+              "value": "https://autopylot.com/oauth/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#e95116",
+    "capabilities": [],
+    "publisher": "AutoPylot",
```

Comment:
Please update two sections as per below. 
  "publisher": "Name of company or organization.",
    "stackOwner": "Name of the owner of the service or product."
Please refer this link https://github.com/microsoft/PowerPlatformConnectors/blob/3ac7cb2ca3a494950d3a6810d2aeeb69f7ccc507/templates/certified-connectors/apiProperties.json

Reply:
updated according to template https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json